vignettes/creating-EML.Rmd

```diff
@@ -417,77 +440,43 @@ for (i in 1:nrow(attribute_table)) {
 
-### Putting the Data Table Together
-Now that we have the `attribute_list`, `physical` information, and `methods`, we 
-can compose the data table, which is the last element we need to create our 
-working EML file. The remaining inputs for the data table can be put in the "data_table"
-tab in the "template.xlsx" excel file. Multiple data tables can be appended to one dataset,
-although it is more common to only have one present. 
-```{r}
-data_table <- read_excel("data-raw/template/template.xlsx", sheet = "data_table")
-for (i in 1:nrow(data_table)) {
-  current <- data_table[i, ]
-  phy <- physical_list[i]
-  att <- attribute_list[i]
-  method <- method_list[i]
-  parent_element <- add_data_table(parent_element = parent_element,
-                                   entity_name = current$entity_name,
-                                   entity_description = current$entity_description,
-                                   physical = phy,
-                                   attribute_list = att,
-                                   methods = method,
-                                   number_of_records = current$number_of_records,
-                                   alternate_identifier = current$alternate_identifier)
-}
-```
-If methods are not present at this level, make sure to take it out. 
-```{r}
-data_table <- read_excel("data-raw/template/template.xlsx", sheet = "data_table")
-for (i in 1:nrow(data_table)) {
-  current <- data_table[i, ]
-  phy <- physical_list[i]
-  att <- attribute_list[i]
-  parent_element <- add_data_table(parent_element = parent_element,
-                                   entity_name = current$entity_name,
-                                   entity_description = current$entity_description,
-                                   physical = phy,
-                                   attribute_list = att,
-                                   number_of_records = current$number_of_records,
-                                   alternate_identifier = current$alternate_identifier)
-}
+## TODO RASTER DATA 
 ```
 
-## Putting It All Together
+## Making the EML document 
 Now that we have all of the items appended to the `parent_element`, we must append it 
-to the "dataset" and also give the file a unique ID number. 
+to the "dataset" and also give the file a unique ID number. The unique id number should 
+match one that you reserved on the EDI data portal. Input this at the top of the 
+document where you define the data tables. 
 ```{r}
-eml <- list(packageId = "uuid::UUIDgenerate()",
-            system = "uuid",
+eml <- list(packageId = edi_number,
+            system = "EDI",
+            access = access,
             dataset = parent_element)
 ```
 We now have all necessary elements to create a working EML document. The final step is to convert it into the correct format. To do so, we can use the EML library's `write_eml` function. 
 ```{r}
-EML::write_eml(eml, "eml.xml")
+EML::write_eml(eml, paste(edi_number, "xml", sep = "."))
```

Comment:
can you specify a safe path for where this will be written? look into the function `?tempfile`

Reply:
Resolved